source/Web/HappyMe.Web/App_GlobalResources/Manage/ViewModels/ChangePasswordViewModel.Designer.cs

```diff
@@ -19,7 +19,7 @@ namespace Resources.Manage.ViewModels {
     // class via a tool like ResGen or Visual Studio.
     // To add or remove a member, edit your .ResX file then rerun ResGen
     // with the /str option, or rebuild your VS project.
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "4.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "17.0.0.0")]
```

Comment:
Are the changes in the resource part of the framework version update?

Reply:
The change is automatic